.github/workflows/ci.yml

```diff
@@ -101,7 +101,7 @@ jobs:
     # https://docs.github.com/en/actions/learn-github-actions/contexts#context-availability
     strategy:
       matrix:
-        msrv: [1.65.0] # 1.65.0 introduced GAT stabilization
```

Comment:
What is the reason for the msrv bump and could you update the comment to reflect that reason

Reply:
Stable once_cell. I will update comment to reflect that